README.md

```diff
@@ -384,19 +384,23 @@ amadeus.booking.hotelBookings.post(
 
 // Points of Interest
 // What are the popular places in Barcelona (based a geo location and a radius)
-amadeus.referenceData.locations.pointsOfInterest.get({
+amadeus.referenceData.locations.pointsOfInterests.get({
```

Comment:
pointsOfInterest without s at the end is the plural

Reply:
I didn't get it - could you clarify?
I am going per the locations endpoint here:
![image](https://user-images.githubusercontent.com/3214885/81977904-b9cefe80-95f8-11ea-98f4-5e2bb812385b.png)

Reply:
I think you are confusing APIs here:
- Airport & City Search has 2 endpoints:
   - amadeus.referenceData.locations.get({ keyword.... -> get all locations from a keyword (autocomplete)
   - amadeus.referenceData.location('ALHR)... -> Retrieve one location based on its ID

The orange one you highlighted on the screenshot is not the same API it is Airport Nearest Relevant where you can retrieve a list of locations of airports around specific geolocation.
---
As discussed POI API has 3 endpoints that should be named exactly as it is in Python

<img width="929" alt="image" src="https://user-images.githubusercontent.com/713173/81991067-b6f7fc00-9640-11ea-95a2-85ab2850b0a5.png">

Reply:
I thought we agreed that __Points of Interest__ endpoints:
![image](https://user-images.githubusercontent.com/3214885/82094367-ee12ef80-96ca-11ea-9f64-99aa67ce0fe1.png)

are structured exactly the same way as __Airport & City Search__ endpoints:
![image](https://user-images.githubusercontent.com/3214885/82094308-c9b71300-96ca-11ea-942d-9e891758cf6f.png)

Hence, they should be implemented similarly:
![image](https://user-images.githubusercontent.com/3214885/82094463-1c90ca80-96cb-11ea-9240-6bf23fec25e0.png)

PS - We also discussed that __Trip Parser__ endpoints are not a good reference structure for __Points of Interest__ endpoints because they (Trip Parser) don't have an overlap at `...tripParserJobs.get()` level:
![image](https://user-images.githubusercontent.com/3214885/82094560-54980d80-96cb-11ea-975e-f61fa0c24bfc.png)